pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -402,6 +405,11 @@ func (o *OpenIDCProvider) getUserInfo(ctx *context.Context, config *v32.OIDCConf
 	if !reflect.DeepEqual(oauth2Token, reusedToken) {
 		o.UpdateToken(reusedToken, userName)
 	}
+	// If an ACR value is configured, validate the value in the claim
+	if config.AcrValue != "" && !validateACR(oauth2Token, config) {
```

Comment:
We can probably skip the check for the emptiness, since it's done in the validate func.

Reply:
done